Developer REST API Backup endpoints (#208)
upstash/docs: devops/developer-api/redis/restore_backup.mdx

---
title: "Restore Backup"

> description: "This endpoint restores a backup for a Redis database."

Reviewer: ```suggestion
description: "This endpoint restores a database data from an existing backup."
```

Reviewer 2: > 

suggestion: "restores data from an existing backup"
